explorer/client/package.json

```diff
@@ -35,7 +35,7 @@
         "react-ace": "^9.5.0",
         "react-dom": "^17.0.2",
         "react-router-dom": "^6.2.1",
-        "sui.js": "file:../../sdk/typescript",
+        "@mysten/sui.js": "file:../../sdk/typescript",
```

Comment:
Right now we still point it to the local repo since we are still iterating on the API. Once the SDK becomes more stable, we will link it to the npm package instead of the local file.